org-roam.el

```diff
@@ -397,6 +397,28 @@ This is equivalent to removing the node from the graph."
   (let ((time (org-read-date nil 'to-time nil "Date:  ")))
     (org-roam--new-file-named (format-time-string "%Y-%m-%d" time))))
 
+(defun org-roam-jump-to-backlink ()
```

Comment:
I was thinking of having the jump be to the exact character at point, but I think this is good enough for me.